dashboard/src/components/Config/PkgRepoList/PkgRepoList.tsx

```diff
@@ -87,13 +99,57 @@ function PkgRepoList() {
   }, [allNS, currentNamespace]);
 
   useEffect(() => {
+    const pluginsConfig = {
+      helmGlobalNamespace: helmGlobalNamespace,
+      carvelGlobalNamespace: carvelGlobalNamespace,
+      kubeappsCluster: kubeappsCluster,
+    } as IConfig;
+
+    Promise.all(
+      configuredPlugins
+        .map(plugin =>
+          PackageRepositoriesService.getRepositoriesPermissions(
+            currentNamespace,
+            pluginsConfig,
+            plugin,
+          ),
+        )
+        .filter(i => i !== undefined),
+    ).then(results => {
```

Comment:
what about using the `await` keyword instead like in other places? 

also, we should have a `.catch` to handle any promise error, shouldn't we?